### ISSUE TITLE
Secure cookies are still saved in the cookie jar

### ISSUE THREAD

**Issue by valentinogagliardi** [curl/curl#5473]
### I did this

`curl -I http://serene-bastion-01422.herokuapp.com/get-secure-cookie/ --cookie-jar -`

### I expected the following

```text
HTTP/1.1 200 OK
Connection: keep-alive
Server: gunicorn/20.0.4
Date: Thu, 28 May 2020 16:56:18 GMT
Content-Type: text/html; charset=utf-8
Content-Length: 30
Set-Cookie: id=3db4adj3d; Secure; Path=/
Via: 1.1 vegur

# Netscape HTTP Cookie File
# https://curl.haxx.se/docs/http-cookies.html
# This file was generated by libcurl! Edit at your own risk.
```

### Instead I've got

```text
HTTP/1.1 200 OK
Connection: keep-alive
Server: gunicorn/20.0.4
Date: Thu, 28 May 2020 17:00:22 GMT
Content-Type: text/html; charset=utf-8
Content-Length: 30
Set-Cookie: id=3db4adj3d; Secure; Path=/
Via: 1.1 vegur

# Netscape HTTP Cookie File
# https://curl.haxx.se/docs/http-cookies.html
# This file was generated by libcurl! Edit at your own risk.

serene-bastion-01422.herokuapp.com	FALSE	/	TRUE	0	id	3db4adj3d
```

### curl/libcurl version

The bug is present at least in:

- curl 7.58.0
- curl 7.61.1

Not present in:

- curl 7.69.1
- curl 7.64.1 (mac os)

### operating system

Linux Ubuntu 18.04.4, Linux CentOS 7

**Comment by bagder** [comment]
Strictly speaking this is not a bug as per RFC 6265 (cookies) as it rather says that this is how it should work.

However, the draft-ietf-httpbis-cookie-alone-01 draft that right now is known as rfc6265bis says that we should "leave secure cookies alone" and that's what curl does since 7a09b52c98ac8d840a8a99, shipped in 7.64.0.

**Comment by bagder** [comment]
Since this issue is already fixed, I'm going to close this issue. There's nothing more for us to do here. Thanks!
